app/data/jsonSchemaForm/emissionIntensityUiSchema.ts

```diff
@@ -0,0 +1,130 @@
+  "ui:order": [
+    "reportDueDate",
+    "submittedDate",
+    "comments",
+    "measurementPeriodStartDate", // aka TEIMP Start Date
+    "measurementPeriodEndDate", // aka TEIMP End Date
+    "emissionFunctionalUnit",
+    "productionFunctionalUnit",
+    "baselineEmissionIntensity",
+    "targetEmissionIntensity",
+    "postProjectEmissionIntensity",
+    "totalLifetimeEmissionReduction",
+    "calculatedEiPerformance", // aka GHG Emission Intensity Performance
+    "adjustedEmissionsIntensityPerformance", // aka GHG Emission Intensity Performance (Adjusted)
+    "dateSentToCsnr",
+    "paymentPercentage", // aka Payment Percentage of Performance Milestone Amount (%)
+    "actualPerformanceMilestoneAmount", // aka Actual Performance Milestone Amount
+    "holdbackAmountToDate", // aka Maximum Performance Milestone Amount
+  ],
+  emissionFunctionalUnit: {
+    "ui:widget": "TextWidget",
+    classNames: "functional-unit",
+  },
+  productionFunctionalUnit: {
+    "ui:widget": "TextWidget",
+    classNames: "functional-unit",
+  },
+  measurementPeriodStartDate: {
+    "ui:widget": "DateWidget",
+  },
+  measurementPeriodEndDate: {
+    "ui:widget": "DateWidget",
+  },
+  baselineEmissionIntensity: {
+    "ui:widget": "NumberWidget",
+    numberOfDecimalPlaces: 8,
+  },
+  targetEmissionIntensity: {
+    "ui:widget": "NumberWidget",
+    numberOfDecimalPlaces: 8,
+  },
+  postProjectEmissionIntensity: {
+    "ui:widget": "NumberWidget",
+    numberOfDecimalPlaces: 8,
+  },
+  totalLifetimeEmissionReduction: {
+    "ui:widget": "NumberWidget",
+    numberOfDecimalPlaces: 8,
+  },
+  adjustedEmissionsIntensityPerformance: {
+    "ui:widget": "AdjustableCalculatedValueWidget",
+    calculatedValueFormContextProperty: "calculatedEiPerformance",
+    isPercentage: true,
+    numberOfDecimalPlaces: 2,
+    hideOptional: true,
+  },
+
+  calculatedEiPerformance: {
+    isPercentage: true,
+    numberOfDecimalPlaces: 2,
+    hideOptional: true,
+  },
+  paymentPercentage: {
+    "ui:widget": "CalculatedValueWidget",
+    isPercentage: true,
+    numberOfDecimalPlaces: 2,
+    hideOptional: true,
+    calculatedValueFormContextProperty:
+      "paymentPercentageOfPerformanceMilestoneAmount",
+  },
+
+  actualPerformanceMilestoneAmount: {
+    "ui:widget": "CalculatedValueWidget",
+    calculatedValueFormContextProperty: "actualPerformanceMilestoneAmount",
+    isMoney: true,
+    hideOptional: true,
+  },
+  holdbackAmountToDate: {
+    "ui:widget": "CalculatedValueWidget",
+    calculatedValueFormContextProperty: "holdbackAmountToDate",
+    isMoney: true,
+    hideOptional: true,
+    renamedId: "maximumPerformanceMilestoneAmount",
+  },
+  dateSentToCsnr: {
+    "ui:widget": "DateWidget",
+  },
+  reportDueDate: {
+    "ui:widget": "DateWidget",
+    isDueDate: true,
+  },
+  submittedDate: {
+    "ui:widget": "DateWidget",
+    isReceivedDate: true,
+  },
+  comments: {
+    "ui:widget": "TextAreaWidget",
+  },
```

Comment:
I deleted these two and they're gone from the db: 
![image](https://github.com/bcgov/cas-cif/assets/77306764/dfd9cd92-d217-4e45-b999-2b2fcbfdf3a1)